domains/vrbz.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "vrbz",
+           "email": "108748279+vrbz@users.noreply.github.com",
+           "discord": "590921984020447235"
```

Comment:
either replace email line with a email which can be contacted or remove the github no reply email